tests/framework/prototype.py

```diff
@@ -186,7 +195,7 @@ def check_rule_sets(self):
 
     def clean_up(self):
         """Cleanup the CR."""
-        raise NotImplementedError
+        pass
```

Comment:
Do we need this function if it just runs `pass`?

Reply:
Hmm, I haven't looked into the other tests that use CREvent, but I think they all call clean_up(). I changed it to `pass` because the new CREvents I wrote don't run clean_up.

Reply:
Actually, I can revert this change.